### PR TITLE
Adds all friendly name audit work to the branch

### PR DESCRIPTION
**Pull request by victoriaatraft**
Adds all friendly name audit work to the branch

@ADPennington, @reitermb and I wanted to bring your attention to the items below. Specifically 59 in the M2.py file as there were some inconsistencies with the code base and the instructions. 
Miles, please feel free to add if I missed anything.

### CODE REVIEW

File: tdrs-backend/tdpservice/parsers/schema_defs/ssp/m2.py [hunk 1]
```diff
@@ -270,7 +270,7 @@
                 Field(
                     item="32A",
                     name='FED_OASDI_PROGRAM',
-                    friendly_name="federal old-age survivors and disability insurance program",
```
Comment (victoriaatraft) [inline tdrs-backend/tdpservice/parsers/schema_defs/ssp/m2.py:282]: The codebook's reference is: Receives Federal Disability Insurance Benefits Under Title II of the Social Security Act (Social Security Disability Insurance [SSDI]).

File: tdrs-backend/tdpservice/parsers/schema_defs/ssp/m2.py [hunk 2]
```diff
@@ -300,7 +300,7 @@
                 Field(
                     item="32D",
                     name='AID_AGED_BLIND',
-                    friendly_name="receives from aid to the aged, blind, and disabled program",
+                    friendly_name="Receives Disability Benefits: Aged, Blind, and Disabled Under Title XVI-AABD",
```
Comment (victoriaatraft) [inline tdrs-backend/tdpservice/parsers/schema_defs/ssp/m2.py]: The codebook says: Code no longer in use, Enter 0

File: tdrs-backend/tdpservice/parsers/schema_defs/ssp/m2.py [hunk 3]
```diff
@@ -750,7 +750,7 @@
                 Field(
```
Comment (victoriaatraft) [inline tdrs-backend/tdpservice/parsers/schema_defs/ssp/m2.py:759]: 59 in particular did not align with the original coding instructions and wanted to bring that to your attention. Comparing to the code book, is no section for E and there are no two A's. Additionally what was in the python file for names, does not correlate with the instructions. I've attached an image for a visual. 

There are similar instances with other files that are [documented in the excel sheet here](https://gorafttech-my.sharepoint.com/:x:/g/personal/vamoroso_teamraft_com/EbOyrZPqlA1Chz8oaVTWDKQBj5CmOYb-N8-U606f1gMvQg?e=TjdlOp).

Amount of Unearned Income m2

**Review by reitermb — APPROVED** [review]
Victoria and I synced up 3/27 to overview—looks great from my perspective, just awaiting pairing to walk through design choices for the wordier edge cases with @ADPennington currently planned for UX sync on 4/2

🚀 🙌

File: tdrs-backend/tdpservice/parsers/schema_defs/tribal_tanf/t4.py [hunk 4]
```diff
@@ -150,7 +150,7 @@
                 Field(
                     item="14",
                     name="BLANK",
```
Comment (ADPennington) [inline tdrs-backend/tdpservice/parsers/schema_defs/tribal_tanf/t4.py]: @elipe17 this `name` here should be `FAMILY_AFFILIATION`. can we change this here or will this introduce problems? cc: @victoriaatraft

File: tdrs-backend/tdpservice/parsers/schema_defs/tribal_tanf/t3.py [hunk 5]
```diff
@@ -448,8 +448,7 @@
         Field(
             item="67",
             name="DATE_OF_BIRTH",
-            friendly_name="date of birth",
-            type="string",
```
Comment (reitermb) [inline tdrs-backend/tdpservice/parsers/schema_defs/tribal_tanf/t3.py:452]: @victoriaatraft looks like the type= line got dropped here, can you add `type="string",` back in alongside the other final tweaks? That should resolve the build errors.

File: tdrs-backend/tdpservice/parsers/schema_defs/ssp/m3.py [hunk 6]
```diff
@@ -557,7 +557,7 @@
         Field(
             item="65B",
             name='RECEIVE_SSI',
-            friendly_name="receives ssi",
+            friendly_name="Receives Disability Benefits: Supplemental Security Income Under Title XVI-SSI or Aged, Blind, and Disabled Under Title XVI-AABD ",
```
Comment (reitermb) [inline tdrs-backend/tdpservice/parsers/schema_defs/ssp/m3.py]: @victoriaatraft one more little tweak, could we remove the extra space at the end of this?

**Comment by ADPennington** [comment]
back in raft review until ready for qasp @victoriaatraft @reitermb

**Comment by reitermb** [comment]
Closing this PR, changes moved to #2931

**Pull request closed without merging**